yb-voyager/cmd/exportData.go

```diff
@@ -274,11 +295,14 @@ func debeziumExportData(ctx context.Context, tableList []*sqlname.SourceName, ta
 		Username:     source.User,
 		Password:     source.Password,
 
-		DatabaseName:      source.DBName,
-		SchemaNames:       source.Schema,
-		TableList:         dbzmTableList,
-		ColumnList:        dbzmColumnList,
-		ColumnSequenceMap: columnSequenceMap,
+		DatabaseName:                source.DBName,
+		SchemaNames:                 source.Schema,
+		TableList:                   dbzmTableList,
+		ColumnList:                  dbzmColumnList,
+		ColumnSequenceMap:           columnSequenceMap,
+		Uri:                         uri,
+		TNSAdmin:                    tnsAdmin,
+		OracleJDBCWalletLocationSet: oracleJDBCWalletLocationIsSet,
```

Comment:
thinking shall we set these props only for oracle db type? Currently looks like common props